ci-operator/step-registry/enable-qe-catalogsource/logging/enable-qe-catalogsource-logging-ref.yaml

```diff
@@ -0,0 +1,20 @@
+ref:
+  as: enable-qe-catalogsource-logging
+  from: upi-installer
+  cli: latest
+  grace_period: 10m
+  commands: enable-qe-catalogsource-logging-commands.sh
+  resources:
+    requests:
+      cpu: 100m
+      memory: 100Mi
+  env:
+  - name: LOGGING_INDEX_IMAGE
+    default: quay.io/openshift-qe-optional-operators/ocp4-index:latest
```

Comment:
It is better to use  quay.io/openshift-qe-optional-operators/aosqe-index:v4.12

Reply:
@anpingli I was checking this index quay.io/openshift-qe-optional-operators/aosqe-index:v4.12 , it doesn't have Loki operator in stable-5.6 channel. It has stable, stable-5.5 and stable-5.7.

Reply:
Performing one rehearsal run before setting the var LOGGING_INDEX_IMAGE to use quay.io/openshift-qe-optional-operators/aosqe-index:v4.12 Will keep the step though enable-qe-catalogsource-logging in case we want to set our own index image in future.

Reply:
@IshwarKanse  Checking .It seems 5.7 blocked 5.6.

Reply:
@IshwarKanse  stable-5.6 are added. I think that was caused by  https://issues.redhat.com/browse/LOG-3521.

Reply:
Added the quay.io/openshift-qe-optional-operators/aosqe-index:v4.12 index for the jobs.